src/main.js

```diff
@@ -90,8 +93,39 @@ function compileTo(src, dest) {
     }
 }
 
+function compileAndRun(src) {
+    var js = compileSource(src);
+    var script = new vm.Script(js);
+    var root = global;
+    root.require = function(id) {
```

Comment:
I think it should be sufficient here to check for whether `id` starts with `.`, meaning it's a local import instead of one managed by npm. Also, paths should be relative to `src`, so this needs to grab the dirname of `src`.